app/assets/stylesheets/main.scss

```diff
@@ -82,6 +82,23 @@
   .grader_distribution {
     margin-bottom: 0;
   }
+
+  #grader-right {
+    height: 380px;
+    overflow: scroll;
+  }
+
+  ::-webkit-scrollbar {
+    -webkit-appearance: none;
```

Comment:
This is fine since it is indeed WebKit-specific. The box-shadow a few lines down should be prefix-free, however.